applications/dashboard/controllers/class.settingscontroller.php

```diff
@@ -1562,6 +1567,7 @@ public function registration($redirectUrl = '') {
         } else {
             // Define some validation rules for the fields being saved
             $configurationModel->Validation->applyRule('Garden.Registration.Method', 'Required');
+            $configurationModel->Validation->applyRule('Garden.Registration.InviteTarget', 'UrlString', 'Invitation Target cannot be numeric, contain special characters or spaces.');
```

Comment:
Shouldn't we wrap validation message with `t()`?
```suggestion
            $configurationModel->Validation->applyRule('Garden.Registration.InviteTarget', 'UrlString', t('Invitation Target cannot be numeric, contain special characters or spaces.'));
```